packages/paste-libraries/syntax-highlighter/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@twilio-paste/syntax-highlighter",
```

issue (blocking): all our library packages should be suffixed with `-library`

```suggestion
  "name": "@twilio-paste/syntax-highlighter-library",
```